roles/monitor/templates/com.tarides.ocluster.monitor.plist

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
> 	<key>KeepAlive</key>

This causes `launchctl` to respawn the script after a delay.  We could add a specific timed schedule, but it seems unnecessary.

Reply:
Got it. Thanks :)

I've not been able to find a doc on how long of a delay we have for `KeepAlive`, but https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html seemed like the most likely place. It only says:

> KeepAlive which indicates that this job needs to be running at all times, rather than the default launch-on-demand behavior, so launchd should always try to keep this job running

Is it possible we might prefer a periodically scheduled job for this, using a periodically scheduled job? https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/ScheduledJobs.html

I feel slightly on easy at the idea of configuring an "always running" script that isn't actually meant to work like a daemon/service, but is instead just constantly restarted. But maybe this is standard practice in macs?